### PR TITLE
 Remove util.StringInSlice because it is defined in containers/common

### PR DESCRIPTION
**Pull request by rhatdan**
[NO NEW TESTS NEEDED] Since this is just a code cleanup  

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM
/lgtm
/hold

File: util/util.go [hunk 1]
```diff
@@ -244,17 +244,6 @@ func Runtime() string {
 	return conf.Engine.OCIRuntime
 }
 
-// StringInSlice returns a boolean indicating if the exact value s is present
-// in the slice slice.
-func StringInSlice(s string, slice []string) bool {
```
Comment (nalind) [inline util/util.go:249]: This is an API break, so it needs to have a wrapper.

File: util/util.go [hunk 2]
```diff
@@ -44,6 +45,11 @@ var (
 	}
 )
 
+// StringInSlice is depracated, use containers/common/pkg/util/StringInSlice
```
Comment (nalind) [inline util/util.go:48]:
```suggestion
// StringInSlice is depracated, use github.com/containers/common/pkg/util.StringInSlice
```

**Comment by nalind** [comment]
Comment nit, otherwise LGTM.

**Comment by TomSweeneyRedHat** [comment]
Fix the nit, then do the rebase, and LGTM

**Review by vrothberg — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
I'm not sure if another rebase is needed or not though.

**Pull request merged**